src/ucs/debug/profile.h

```diff
@@ -0,0 +1,344 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2016.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifndef UCS_PROFILE_H_
+#define UCS_PROFILE_H_
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+
+#include <ucs/sys/preprocessor.h>
+#include <ucs/time/time.h>
+#include <ucs/debug/log.h>
+
+
+#define UCS_PROFILE_STACK_MAX 64
+
+
+/**
+ * Profiling modes
+ */
+enum {
+    UCS_PROFILE_MODE_ACCUM, /* Accumulate elapsed time per location */
+    UCS_PROFILE_MODE_LOG,   /* Record all events */
+    UCS_PROFILE_MODE_LAST
+};
+
+
+/**
+ * Profiling location type
+ */
+typedef enum {
+    UCS_PROFILE_TYPE_SAMPLE,        /* Sample only */
```

Comment:
If you already use doxygen style comments for the struct description, why not to use /**< */ for the fields ?